test/util.go

```diff
@@ -63,23 +63,23 @@ type context struct {
 	kubeclient *kubernetes.Clientset
 	karclient  *versioned.Clientset
 
-	namespace string
-	queues    []string
+	namespaces []string
+	queues     []string
 }
 
 func splictJobName(cxt *context, jn string) (string, string) {
 	nss := strings.Split(jn, "/")
 	if len(nss) == 1 {
-		return cxt.namespace, nss[0]
+		return cxt.namespaces[0], nss[0]
 	}
 
 	return nss[0], nss[1]
 }
 
 func initTestContext() *context {
 	cxt := &context{
-		namespace: "test",
-		queues:    []string{"q1", "q2", "test"},
+		namespaces: []string{"test", "n1", "n2"},
```

Comment:
Oh, how about have a parameters to create q1, q2 as namespace or Queue? We can have a random value for `enable-namespace-as-queue` to improve the coverage.

Reply:
env is ok to me :)